hw/xbox/dsp/dsp_dma.c

```diff
@@ -137,9 +179,51 @@ static void dsp_dma_run(DSPDMAState *s)
             assert(false);
         }
 
+#ifdef DEBUG
+        char dsp_space_name = '?';
```

Comment:
nit: There might be 3 DSP spaces of relevance:

* DSP space of current DMA block (currently assumed as 'x')
* DSP space of next DMA block (currently assumed as 'x')
* DSP space of transfer DSP buffer

The variable name therefore might be confusing in the future.